pkg/server/routes.go

```diff
@@ -292,6 +292,15 @@ func (s *Server) routes(
 				cedarLdapClient.FetchUserInfo,
 				emailClient.SendSystemIntakeReviewEmail,
 			),
+			services.NewGRTReviewSystemIntake(
+				serviceConfig,
+				models.SystemIntakeStatusSUBMITDRAFTBIZCASE,
+				store.UpdateSystemIntake,
+				services.NewAuthorizeRequireGRTJobCode(),
```

Comment:
We don't want to require the GRT job code for this action, since it will be done by the requester.